_ols-1/projects-participants.md

```diff
@@ -13,7 +13,7 @@ photos:
 
 {% assign all-participants = '' %}
 {% for project in projects %}
-    {% assign p-pparticipants = '' %}
```

Comment:
It was not really a spelling mistake, mostly variable names

Reply:
Okay. Thanks